### ISSUE TITLE
The keyboard focus is not set to the file dialog

### ISSUE THREAD

**Issue by kryksyh** [audacity/audacity#9309]
### Bug description

The full description: https://github.com/musescore/MuseScore/issues/13113

The solution used in MuseScore is to force GTK3 dialog, which is causing problems in Audacity: #9308 

### Steps to reproduce

- Open the app
- Open file dialog
- Try to use keyboard arrows

### Expected behavior

The keyboard focus should be set to file open dialog

### Actual behavior

The keyboard focus is left on the main window

### Audacity Version

fe4ca159

### Operating system

Linux

**Comment by chinakov** [comment]
This is still present on the latest Master (47ecf3eeabc46945542e87107355900f1fd5c326)

**Comment by chinakov** [comment]
This is fixed, thanks!
